great-idea/css/services.css

```diff
@@ -0,0 +1,213 @@
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
+
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+.container {
+	margin: 0 auto;
+	max-width: 960px;
+}
+
+/* Nav bar & Logo */
+.line1 {
+	display: flex;
+	/* border: 1px solid red; */
+}
+
+/* header */
+.buttons {
+	margin: 20px 20px 20px 0;
+	width: 75%;
+	justify-content: space-between;
+	display: flex;
+	/* border: 1px solid blue; */
+}
+
+.services-header-img {
+    width: 100%;
+}
+
+a {
+	justify-content: space-between;
+	text-decoration: none;
+	color: lightgray;
+	/* border: 1px solid blue; */
+}
+
+.logo {
+	margin: 10px 20px 20px 20px;
+	display: flex;
+    width: 100%;
+	justify-content: flex-end;
+	/* border: 1px solid blue; */
+}
+/* header */
+
+
+
+/* content / boxes */
+
+.services h2 {
+    font-size: 25px;
+    margin-top: 25px;
+
+}
+
+.services p {
+    border-bottom: 1px solid black;
+    margin-bottom: 25px;
+    height: 80px;
+}
+
+/* .content {
+    border: 1px solid red;
+} */
+
+.boxes {
+    display: flex;
+    flex-wrap: wrap;
+    flex-direction: row;
+    border-bottom: 2px solid black;
+    justify-content: space-between;
+    width: 100%;
+}
+
+.box1, .box2, .box3, .box4, .box5, .box6 {
+    display: flex;
+    border: solid 1px black;
+    background-color: lightcyan;
+    flex-direction: row;
+    flex-wrap: wrap;
+    height: 250px;
+    width: 48%;
+    justify-content: space-between;
+    /* margin-right: 30px; */
+    margin-bottom: 30px;
+    padding: 20px;
+}
+
+.more {
+    display: flex;
+    align-items: center;
+    font-size: 15px;
+    border: 1px solid black;
+    border-radius: 8px;
+    padding: 2px 55px 2px 55px;
+    background-color: white;
+}
+/* content / boxes */
+
+
+/* some facts / footer */
+
+.facts {
+    display: flex;
+    flex-wrap: wrap;
+    /* border: 1px solid red; */
+    justify-content: space-between;
+    margin-top: 25px;
+}
+
+.other-facts {
+    flex-wrap: wrap;
+    /* border: 1px solid black; */
+    width: 50%;
+    height: 50px;
+}
+
+.services-info-img {
+    display: flex;
+    justify-content: flex-end;
+}
+
+.facts p {
+    flex-wrap: wrap;
+    /* border: 1px solid blue; */
+    width: 50%;
+    position: relative;
+    bottom: 250px;
+}
+
+ul {
+    position: relative;
+    right: 828px;
+    bottom: 140px;
+    list-style-type: disc;
+    list-style-position: inside;
+}
+
```

Comment:
```suggestion
ul {
    position: relative;
    right: 828px;
    bottom: 140px;
    list-style-type: disc;
    list-style-position: inside;
    margin-top: 25px;
}

ul li {
    padding: 10px;
}
```

Reply:
Maybe something like this? Don't give up on a good project when you're right at the end!